homework/src/index.js

```diff
@@ -1,47 +1,121 @@
-'use strict';
+"use strict";
 
 {
-  function fetchJSON(url, cb) {
-    const xhr = new XMLHttpRequest();
-    xhr.open('GET', url);
-    xhr.responseType = 'json';
-    xhr.onload = () => {
-      if (xhr.status < 400) {
-        cb(null, xhr.response);
-      } else {
-        cb(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
-      }
+    function createAndAppend(name, parent, options = {}) {
+        const elem = document.createElement(name);
+        parent.appendChild(elem);
+        Object.keys(options).forEach((key) => {
+            const value = options[key];
+            if (key === "text") {
+                elem.innerHTML = value;
+            } else {
+                elem.setAttribute(key, value);
+            }
+        });
+        return elem;
+    }
+
+    function renderRepo(container, repository, linkName) {
+
+        container.innerHTML = "";
+        const table = createAndAppend("table", container, { class: "table" });
+        const tr = createAndAppend("tr", table);
+        const td = createAndAppend("td", tr, { text: "<b>Repository: </b> " });
+
+        createAndAppend("a", td, {
+            text: repository.html_url,
+            text: linkName,
```

Comment:
There is an ESLint warning about a duplicate `text` property. I guess the second one takes precedence over the first one. Nevertheless, something to correct.